.gitignore

```diff
@@ -1,5 +1,6 @@
 # Generic Files
 *.txt
 node_modules
-**/contentBundle.js
-**/contentBundle.js.map
\ No newline at end of file
+dist
+!dist/manifest.json
+!dist/popup/*
```

Comment:
whats this with the exclaimation?

Reply:
The exclamation tells gitignore to _not exclude_ those files / folders.

Reply:
ok thanks